lua/fishing_mod/sv_init.lua

```diff
@@ -53,37 +53,43 @@ hook.Add("CanTool", "Fishingmod:CanTool", function(ply, trace, tool)
 		return false		
 	end
 end)
+fishingmod.lastBaitSpawn = CurTime()
 
 concommand.Add("fishing_mod_buy_bait", function(ply, command, arguments)
-	local type = table.concat(arguments, " ")
-	local rod = ply:GetFishingRod()
-	if not rod then return end
+	if CurTime() >= fishingmod.lastBaitSpawn + 0.625 then

```

Comment:
Put the delay between each bait spawn as a constant variable somewhere so the reader knows what that magical number means